package.json

```diff
@@ -18,9 +18,11 @@
     "@docusaurus/preset-classic": "^3.4.0",
     "@mdx-js/react": "^3.0.1",
     "clsx": "^2.1.1",
+    "d3": "^7.9.0",
     "prism-react-renderer": "^2.3.1",
     "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "react-dom": "^18.3.1",
+    "venn.js": "^0.2.20"
```

Comment:
@siarhei-leuchanka Well done; I see you went above and beyond with this! 👍 

I am a bit hesitant to accept this change, though, as this dependency [doesn't seem to be very maintained](https://github.com/benfred/venn.js). Additionally, I am trying to keep the maintenance cost of this project to a minimum, and I wouldn't like to maintain custom React components **if not needed**.

Have you considered using Draw.io, as we are doing it in other places? For example: https://github.com/rossumai/university/tree/aaf5f0343db2853e6fec98fe76c854710b7fdb73/docs/learn/sandboxes/_img

Reply:
@mrtnzlml sure, did not spot it. I will redo it with draw.io